### DISCUSSION TITLE
Failed to create a stack on AWS

### DISCUSSION THREAD

**Discussion by hungleon2112** [mozilla/hubs#4805] [Hubs Cloud Questions] [answered]
Hello, today I tried to create a stack on AWS. I followed the instruction(both documentation and video on youtube), but no luck for me. I got this message as the very first error. Can someone help me to figure it out?
Thank you!
![image](https://user-images.githubusercontent.com/10544674/140102343-7b772c9c-e4eb-47db-8b07-f0f1a0f92d94.png)

**Comment by pattersonbl2** [comment, marked as answer]
Do you know what version of the Hubs cloud you tried to launch ?

**Reply by hungleon2112** [reply]
hi @Pattersonbl2 , thank you for your reply. I'm using version 1.1.2.

Here is the screenshot
![image](https://user-images.githubusercontent.com/10544674/140358191-1d40b05b-7614-4013-9381-a464f9db13bc.png)

And this is the errors I get
![image](https://user-images.githubusercontent.com/10544674/140358518-6d70dbc5-690e-4ffe-b82e-f2688d39bb44.png)

**Reply by pattersonbl2** [reply]
We should have a new template for hubs cloud in by the end of day.

**Reply by hungleon2112** [reply]
hi @Pattersonbl2 , sorry if I bother you, but may I ask do we have the fix yet? I've checked and tried to re-install but still get the same errors.

**Reply by vjFred** [reply]
Same here, I just try on a new stack and this error is still there, any news o maybe can we upload the template from a file ?

**Reply by vjFred** [reply]
I just try using this .yaml file but it results in a Region error,
https://github.com/mozilla/hubs-ops/blob/master/cloudformation/stack.yaml
am I using it correctly?

**Reply by RBOSAMIYA9** [reply]
@vjFred  are you able to install it?

**Reply by vjFred** [reply]
@RBOSAMIYA9 No, I download it and make an  .yaml file but it gives me a Region error.  have you tried it?

**Reply by pattersonbl2** [reply]
The marketplace template has been patched and ready for people to use. I would redeploy any stacks using this due to updates that may come in the future.

**Reply by vjFred** [reply]
Hi Bandon, the template has deployed ok, however, thumbnails for avatars in the avatar settings for each user, and scenes are not shown 
![image](https://user-images.githubusercontent.com/69004885/142741385-14177c51-8ca3-4a00-8f59-17082470c677.png)

How can we solve this, especially for the users that can not check the preview of their avatar selection?
